contribs/gmf/examples/search.html

```diff
@@ -0,0 +1,112 @@
+    <link rel="stylesheet" href="../../../node_modules/bootstrap/dist/css/bootstrap.css" type="text/css">
+    <style>
+      gmf-map > div {
+        width: 600px;
+        height: 400px;
+      }
+      /* CSS stolen from https://github.com/bassjobsen/typeahead.js-bootstrap-css/ */
+      span.twitter-typeahead .tt-menu {
+        position: absolute;
+        top: 100%;
+        left: 0;
+        z-index: 1000;
+        display: none;
+        float: left;
+        min-width: 160px;
+        padding: 5px 0;
+        margin: 2px 0 0;
+        list-style: none;
+        font-size: 14px;
+        text-align: left;
+        background-color: #ffffff;
+        border: 1px solid #cccccc;
+        border: 1px solid rgba(0, 0, 0, 0.15);
+        border-radius: 4px;
+        -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+        box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+        background-clip: padding-box;
+      }
+      span.twitter-typeahead .header {
+       background-color: #CCDDEE;
+       padding: 4px 0;
+      }
+      span.twitter-typeahead .tt-suggestion {
+        display: block;
+        padding: 3px 20px;
+        margin-bottom: 0px;
+        clear: both;
+        font-weight: normal;
+        line-height: 1.42857143;
+        color: #333333;
+        white-space: nowrap;
+      }
+      span.twitter-typeahead .tt-suggestion:hover,
+      span.twitter-typeahead .tt-suggestion:focus {
+        color: #ffffff;
+        text-decoration: none;
+        outline: 0;
+        background-color: #428bca;
+      }
+      span.twitter-typeahead .tt-suggestion.tt-cursor {
+        color: #ffffff;
+        background-color: #428bca;
+      }
+      span.twitter-typeahead .tt-suggestion button {
+        background: none;
+        border: 0;
+        float: right;
+      }
+      span.twitter-typeahead {
+        width: 100%;
+      }
+      .input-group span.twitter-typeahead {
+        display: block !important;
+      }
+      .input-group span.twitter-typeahead .tt-menu {
+        top: 32px !important;
+      }
+      .input-group.input-group-lg span.twitter-typeahead .tt-menu {
+        top: 44px !important;
+      }
+      .input-group.input-group-sm span.twitter-typeahead .tt-menu {
+        top: 28px !important;
+      }
+    </style>
+  </head>
+  <body ng-controller="MainController as ctrl">
+    <div class="container-fluid">
+      <gmf-map gmf-map-map="ctrl.map"></gmf-map>
+      <p>Search for « Laus » for example…<p>
+      <gmf-search 
+        gmf-search-map="ctrl.map"
+        gmf-search-datasources='[{
+            dataTitle: "From demo 1.6",
+            url: "http://geomapfish-demo.camptocamp.net/1.6/wsgi/fulltextsearch",
+            projection: "EPSG:21781"
+          }, {
+            dataTitle: "From Cartoriviera - ",
+            url: "http://map.cartoriviera.ch/main/wsgi/fulltextsearch",
+            projection: "EPSG:21781",
+            groupByProperties: ["04rue", "03adresse"]
+          }]'>
```

Comment:
I don't think this is good practice to define such config objects in the HTML. Instead, this config object should be defined on the main controller's scope.